firebase-perf/src/main/java/com/google/firebase/perf/logging/ConsoleUrlGenerator.java

```diff
@@ -0,0 +1,58 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.firebase.perf.logging;
+
+public final class ConsoleUrlGenerator {
+  private static final String URL_BASE_PATH = "https://console.firebase.google.com";
+  private static final String UTM_MEDIUM = "perf-android-studio";
+
+  /**
+   * Generate the console URL for Firebase Performance dashboard page.
+   *
+   * @param projectId the Firebase project ID
+   * @param packageName the name of this application's package
+   */
+  public static String generateDashboardUrl(String projectId, String packageName) {
+    return String.format(
+        "%s/project/%s/performance/app/android:%s/trends?utm_medium=%s",
```

Comment:
Just a thought, can we fetch this from remote config? And have this value as default value on client if RC does not have any value?

That way we are not worried about hardcoding the URL path on the client?

Reply:
My two cents:
1. I think it would be good if RC can have this value but RC might be an overkill. Values in RC should be expected to change over time so that we can remotely control the value. However, I do not expect the URL path to be changed very often. 
2. If the URL path gets changed, the FE should handle the redirection from the old URL to the new URL at least for a while.

Reply:
Sounds good. Lets keep this aside for now and update SDKs when we change the endpoint in future.